main.go

```diff
@@ -153,7 +153,8 @@ func main() {
 	conf := cp.GetConfig()
 
 	setupLog.Info("Using adapter", "image", conf.StorageHelperImage.TaggedImage())
-	setupLog.Info("Using model-mesh", "image", conf.ModelMeshImage.TaggedImage())
+	setupLog.Info("Using modelmesh", "image", conf.ModelMeshImage.TaggedImage())
+	setupLog.Info("Using modelmesh REST proxy", "image", conf.RESTProxy.Image.TaggedImage())
```

Comment:
Maybe only log this if the proxy is enabled, and otherwise an info message that it's disabled?

Reply:
Yea, that makes sense. Updated.